frontend/src/components/ApplicationNameBar.tsx

```diff
@@ -32,21 +33,10 @@ const ApplicationNameBar: FC<ApplicationNameBarProps> = ({ text, href, checklist
             <div className='space-x-3 md:space-x-6'>
               <Link
               href={checklistUrl}
-              className='text-white bg-blue-dark rounded-xl py-2 px-3 font-bold'
+              className='text-white visited:text-white bg-aqua-dark rounded-xl py-2 px-3 font-bold align-middle'
               >
-              {checklist}
+              <BookmarkBorderIcon className=' ' />&nbsp;{checklist}
```

Comment:
```suggestion
              <BookmarkBorderIcon />&nbsp;{checklist}
```

(is the empty css class necessary?)

Reply:
nah. I just missed removing it.